admin/create-theme/create-theme-form.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+require_once( __DIR__ . '/theme-tags.php' );
+
+class Create_Theme_Form {
```

Comment:
This is unusual at least as a name for a class to be named like an action. Maybe `Theme_Form` would suffice?

Reply:
Makes sense. Renamed the class name. 👍